.gitignore

```diff
@@ -93,3 +93,7 @@ mockserver-proxy-war/tomcat/
 mockserver-proxy-war/tomcat_proxy/
 mockserver-war/tomcat/
 mockserver-war/tomcat_mockserver/
+
+# ignored files #
+##############################
+*IGNORE*
```

Comment:
I often add this to my repos so I can add environment config files that should be ignored by Git. I am happy to remove this if you prefer.